org_chart.js

```diff
@@ -1,73 +1,138 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
-}
-
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
-  }
-]
-
-// assign each a manager
-let [alan, grace, donald] = new_hires
 
-alan.manager = elon
-grace.manager = elon
-donald.manager = elon
+function Employee(name, email, position, project){
+  this.name = name
+  this.email = email
+  this.position = position
+  this.skills = []
+  this.description = `${name} is a ${position} working on ${project}. He can be reached at ${email}`
+}
 
-// send to training
-if (!alan.skills) {
-  alan.skills = []
+Employee.prototype.set_manager = function(manager){
+  this.manager = manager
 }
-alan.skills.push("Mathematics")
 
-if (!grace.skills) {
-  grace.skills = []
+Employee.prototype.set_project = function(project){
+  this.project = project
 }
-grace.skills.push("Cobol")
 
-if (!donald.skills) {
-  donald.skills = []
+Employee.prototype.add_skill = function (skill){
+  this.skills.push(skill)
 }
-donald.skills.push("Computational Complexity")
+
+let elon = new Employee("Elon Musk", "elon.musk@musky.com", "Manager")
+
+let alan = new Employee("Alan Turing", "alan.turing@musky.com", "Computer Scientist", "Cryptography")
+alan.add_skill("Mathematics")
+alan.set_manager(elon)
+
+let grace = new Employee("Grace Hopper", "grace.hopper@musky.com", "Computer Scientist", "Compilers")
+grace.add_skill("Cobol")
+grace.set_manager(elon)
+
+let donald = new Employee("Donald Knuth", "donald.knuth@musky.com", "Mathematician", "Algorithm Analysis")
+donald.add_skill("Computational complexity")
+donald.set_manager(elon)
```

Comment:
To reduce duplication further, you could add a function that set's the manager, adds the skill, and prints the person's description